xmtp_mls/src/groups/mod.rs

```diff
@@ -272,6 +272,15 @@ where
         account_addresses_to_add: Vec<String>,
     ) -> Result<(), GroupError> {
```

Comment:
I only added this to add_members, since that's what's exposed to sdks, and add_members_by_installation_id is used during syncing the group members
I didn't want a sync to perpetually fail if somehow more than the limit were able to be added

Reply:
That makes a lot of sense, good thinking. Enforcing the 250 limit on users rather than installations is also easier to explain to developers and end users